test/library/standard/IO/ioHintSetOperations.chpl

```diff
@@ -0,0 +1,24 @@
+use IO;
+use Set;
+
+// constructors work
+const empty = ioHintSet.empty;
+const seq = ioHintSet.sequential;
+const rand = ioHintSet.random;
+const pref = ioHintSet.prefetch;
+const mmap = ioHintSet.mmap;
+
+// set operations work
+writeln(empty == seq & rand & pref & mmap);
+writeln(seq | rand != pref | mmap);
+writeln(seq != seq | rand);
+writeln(pref & pref | rand == pref | rand);
```

Comment:
I think it would be a bit more effective if the checks were more isolated.  For instance, only checking one `==` or `&` in a single writeln.  Right now if something does go wrong with one of them, it'll be harder to nail down which operation is having problems and I think some of them can hide problems with others